components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanCrashReporterService.kt

```diff
@@ -52,21 +52,12 @@ class GleanCrashReporterService(
                 return@run
             }
 
-            if (!Glean.isInitialized()) {
-                logger.error("Glean is not initialized, crash reporter service cannot record metrics")
-                return@run
-            }
-
             if (!checkFileConditions()) {
                 // checkFileConditions() internally logs error conditions
                 return@run
             }
 
-            // Parse the persisted crashes
             parseCrashFile()
-
-            // Clear persisted counts by deleting the file
-            file.delete()
```

Comment:
Is there a specific reason why you're moving this?

Reply:
No, it was an artifact of the first commit before I made the queueing system internal to glean.  I was passing the `parseCrashFile()` function to Glean to execute as part of it's init and had moved the `file.delete()` there to ensure it wasn't deleted before it was read.